src/bin/cargo/commands/owner.rs

```diff
@@ -7,24 +7,71 @@ pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
         .arg_quiet()
-        .arg(Arg::new("crate").action(ArgAction::Set))
+        .arg(Arg::new("crate"))
```

Comment:
Shouldn't this be down below with the flags and be hidden as well?

Reply:
Should be hidden? Sorry, it didn't seem to be hidden before, I didn't understand, but I did it anyway.

Reply:
Its hidden now but still not moved down to the other related args

Reply:
Why move it below the other args? Isn't this the best location?

Reply:
All of the arguments on the `owner` command should be grouped together for discoverability purposes.

Reply:
Sorry, I have something to discuss. Because of the existence of ```--add```、```--remove``` and ```--list```, if ```.arg(Arg::new("crate").hide(true))```  is moved to a subcommand, an error will occur.

Reply:
I'm not asking that it get moved into a subcommand but just before the definition of `--add`.

Reply:
Just move location? Hope it's what you want.